src/frontend/package.json

```diff
@@ -12,10 +12,15 @@
   "license": "MIT",
   "dependencies": {
     "@blueprintjs/core": "^1.6.0",
+    "@types/blueimp-md5": "^1.1.30",
+    "@types/jquery": "^2.0.40",
+    "@types/jquery.cookie": "^1.4.28",
```

Comment:
it would be nice if we didn't need jquery -- can you try using js-cookie?  It's a bit more lightweight.
https://github.com/js-cookie/js-cookie